src/main/java/com/google/devtools/build/lib/sandbox/LinuxSandboxedSpawnRunner.java

```diff
@@ -95,11 +110,43 @@ public static boolean isSupported(final CommandEnvironment cmdEnv) throws Interr
     return isSupported;
   }
 
-  private static boolean computeIsSupported(CommandEnvironment cmdEnv, Path linuxSandbox)
+  private static SupportLevel computeIsSupported(CommandEnvironment cmdEnv, Path linuxSandbox)
       throws InterruptedException {
+    try {
+      runBinTrue(cmdEnv, linuxSandbox, /* wrapInBusybox= */ false);
+      return SupportLevel.SUPPORTED;
```

Comment:
Should the fact it's using busybox be logged or informed to the user at all somehow?

Reply:
That's a good question. I don't think that launching a `busybox` shell would have relevant overhead compared to setting up the sandbox, but I can run a benchmark to confirm that there is no regression. If so, there's probably no reason to tell the user about this.

Reply:
Probably worth logging anyways since your other comment alludes that this circumvention will be going away.
Would be a good warning to let users know they should get ahead of the fix.